quickwit/quickwit-jaeger/src/integration_tests.rs

```diff
@@ -226,7 +230,47 @@ async fn test_otel_jaeger_integration() {
             .unwrap()
             .into_inner();
         assert_eq!(find_trace_ids_response.trace_ids.len(), 1);
+        assert_eq!(find_trace_ids_response.trace_ids[0], [5; 16]);
+
+        // Search traces with an error.
+        let query = TraceQueryParameters {
+            service_name: "quickwit".to_string(),
+            operation_name: "list_splits".to_string(),
+            tags: HashMap::from([("error".to_string(), "true".to_string())]),
+            start_time_min: None,
+            start_time_max: None,
+            duration_min: None,
+            duration_max: None,
+            num_traces: 10,
+        };
+        let find_trace_ids_request = FindTraceIDsRequest { query: Some(query) };
+        let find_trace_ids_response = jaeger_service
+            .find_trace_i_ds(tonic::Request::new(find_trace_ids_request))
```

Comment:
is `find_trace_i_ds` the right name? it seems like `find_trace_ids` is.

Reply:
codegen